internal/observer/observer.go

```diff
@@ -26,7 +26,7 @@ func Run(dt string) {
 	for {
 		switch dt {
 		case "substrate":
-			conn := &recws.RecConn{KeepAliveTimeout: 5 * time.Second, WriteTimeout: time.Second * 5, ReadTimeout: 60 * time.Second}
+			conn := &recws.RecConn{KeepAliveTimeout: 5 * time.Second, WriteTimeout: time.Second * 5, ReadTimeout: 20 * time.Second}
```

Comment:
I've been getting a lot of intermittent connection errors with this build and changing this back seemed to help. I think we should keep it at 60.

Reply:
Done!